docs/docker.md

```diff
@@ -2,55 +2,103 @@
-`admin-party-mode` (no user). This command exposes couchdb on the port `5984`.
+You need docker and using docker-compose is a good idea. Here are the versions used for develoment
 
-```bash
-$ docker run -d \
-    --name cozy-stack-couch \
-    -p 5984:5984 \
-    -v $HOME/.cozy-stack-couch:/opt/couchdb/data \
-    apache/couchdb:2.3
-$ curl -X PUT http://127.0.0.1:5984/{_users,_replicator}
 ```
+➜  docker git:(master) docker -v
+Docker version 18.09.5, build e8ff056
+➜  docker git:(master) docker-compose version
+docker-compose version 1.24.0, build 0aa5906
+docker-py version: 3.7.2
+CPython version: 3.6.7
+OpenSSL version: OpenSSL 1.1.1  11 Sep 2018
+```
+
+Cozy Stack needs at least a CouchDB 2.3 to run.
+
+A reverse proxy is strongly recommended for HTTPS, Caddy being a good pick since it's able to provide on-demand TLS. Though any other can do.
+
+Redis for caching is optional.
+
+## Supported tags
+
+- `development`. Running `cozy-stack` compiled against `master` branch in `development` mode, a.k.a without enforcing HTTPS and other security features, so it can be run localy without certificates. Plus `couchdb` and `mailhog`, so it's a all in one environment for developers.
 
-Verify your installation at: http://127.0.0.1:5984/_utils/#verifyinstall.
+- `latest`. Running `cozy-stack` only compiled against `master` branch in `production` mode. *Use this one*.
 
-Note: for running some unit tests, you will need to use `--net=host` instead of
-`-p 5984:5984` as we are using CouchDB replications and CouchDB will need to be
-able to open a connexion to the stack.
+- `<version>` tag. Each one match a source code tag and are compiled in production mode, enforcing HTTPS and so on. Use this one.
+- `latest` tag. It matches the `master` branch of the source code, also compiled in production mode. Can be unstable, but you'll have the latest features here.
+- `development`. It also matches the `master` branch of the source code, but it's compiled in development mode, hence not enforcing HTTPS so it can be used localy without settings up
 
-## Building a cozy-stack _via_ Docker
+## The secure and easy way
 
-Warning, this command will build a linux binary. Use
-[`GOOS` and `GOARCH`](https://golang.org/doc/install/source#environment) to
-adapt to your own system.
+The production image enforce security, but can be difficult to use since it requires setting up a CouchDB, a reverse proxy, getting a certificate. So we provide an easy setup using docker-compose that will get a Cozy Stack + CouchDB database + Redis cache + Caddy reverse proxy started.
+
+Here's the [docker-compose.yml](https://raw.githubusercontent.com/cozy/cozy-stack/master/docker/docker-compose.yml) which rely on an `.env` file for parameters, a [sample is here](https://raw.githubusercontent.com/cozy/cozy-stack/master/docker/env.sample). On your server you need docker & docker-compose installed, plus a `*.your.domain` DNS record.
```

Comment:
The server also needs to be reachable from Internet (we had several people who have tried to install cozy on a server inside a LAN).

Reply:
Ok